assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -123,6 +137,20 @@ export const reducer = ( state, { type, payload } ) => {
 			};
 		}
 
+		case RESET_ACCOUNTS: {
+			return {
+				...state,
+				accounts: undefined,
+				settings: {
+					...state.settings,
+					accountID: undefined,
+					propertyID: undefined,
+					internalWebPropertyID: undefined,
+					profileID: undefined,
```

Comment:
I think a single action is fine here, but if "-Accounts" sounds off, then we could have something more generic like `reload` action which sets the whole store back to its `INITIAL_STATE` ? It might be worth triggering existing tag requests again, and resetting all the resolvers too. Basically like a page reload, without the reload 😄 
I thought about using `reset` but our reset action actually resets data on the server too, which is not what we want here.